hoster/common_test.go

```diff
@@ -16,3 +19,17 @@ func fakeCliContext(t *testing.T, args []string) *cli.Context {
 	}
 	return cli.NewContext(app, set, nil)
 }
+
+func captureOutput(t *testing.T, f func()) string {
+	rescueStdout := os.Stdout
+	r, w, err := os.Pipe()
+	assert.NoError(t, err)
+	os.Stdout = w
+	f()
+	err = w.Close()
+	assert.NoError(t, err)
+	out, err := ioutil.ReadAll(r)
+	assert.NoError(t, err)
+	os.Stdout = rescueStdout
```

Comment:
Nice, `captureOutput` seem to be quite verbose and easy to read

Reply:
Yes. Go is verbose. Sometimes too much, but it is fun how different it is :).